GREEDY/OCRs/TesseractOcr.cs

```diff
@@ -7,20 +7,16 @@ namespace GREEDY.OCRs
 {
     public class TesseractOcr : IOcr
     {
-        private readonly TesseractEngine _tesseract;
+        private TesseractEngine _tesseract;
 
-        public TesseractOcr()
+        public Receipt ConvertImage(Bitmap image)
         {
             _tesseract = new TesseractEngine
             (
                 Environments.AppConfig.TesseractDataPath,
                 Environments.AppConfig.OcrLanguage,
-                EngineMode.TesseractAndCube
+                EngineMode.TesseractOnly
             );
-        }
-
-        public Receipt ConvertImage(Bitmap image)
-        {
             var page = _tesseract.Process(image);
             return page.GetReceipt();
         }
```

Comment:
why did you merge this methods to one? both OCR class was coded with the same style.

Reply:
Because there was a problem with tesseract when trying to OCR a second time, it would throw an exception that tesseract is already in progress, so I moved the initalization of tesseract from the contructor to the method (so it would be called seperatly every time). Maybe we could find another fix later on.

Reply:
I get it. Thanks 😇